inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -810,24 +819,33 @@ void MKLDNNGraph::Infer(MKLDNNInferRequest* request, int batch) {
 
     ENABLE_CPU_DEBUG_CAP(NodeDumper nd(config.debugCaps, infer_count));
 
-    for (int i = 0; i < graphNodes.size(); i++) {
-        if (request != nullptr) {
+    for (int i = 0; i < constantGraphNodes.size(); i++) {
+        if (request != nullptr)
+            request->ThrowIfCanceled();
+
+        if (batch > 0)
+            constantGraphNodes[i]->setDynamicBatchLim(batch);
+
+        ENABLE_CPU_DEBUG_CAP(nd.dumpOutputBlobs(constantGraphNodes[i]));
```

Comment:
I think we should also dump input  blobs for the constant nodes as there might be intermediate layers on a constant path such as reorder, reshape etc.

Reply:
Yes, I added, thanks

Reply:
Great! But why do we need this check `if (request != nullptr)` ?